### ISSUE TITLE
Make the app available in the more countries (e.g. the same as the iOS pendant)

### ISSUE THREAD

**Issue by dekarl** [cyclestreets/android#137]
After seeing #46 and using the app on iOS I tried to install it on an Android device, but it is not available in my country (Germany) on Google Play.

**Comment by jasontrublu** [comment]
important for me as well, I live in London, but am still registered in German app store (don't ask, stuck in contract, but great roaming options)

**Comment by jf1** [comment]
Can/do both of you install manually from http://www.jezuk.co.uk/cyclestreets/master/CycleStreets-vNext-latest.apk

**Comment by mvl22** [comment]
@jf1 That is not a public URL at present. We will have a proper download area set up soon.

**Comment by jf1** [comment]
Understood, and I wouldn't have posted it in a more public area but thought it was considered ok here as there are other links to those downloads within this project's Github issues. (That's how I was reminded of the location!)

**Comment by mvl22** [comment]
@jf1 OK, I will go and delete those too. The dev download area represents releases no quality control so people shouldn't be pointed publicly to it.

**Comment by dekarl** [comment]
@jf1 thanks for the hint, manual downloads are a nice way to get other developer-type-persons started, but these builds leave joe average behind with manual update nightmare as the play store doesn't know about it.
@mvl22 Over at the MythTV Player for Android we use [Fastlane's supply](https://github.com/fastlane/fastlane/tree/master/supply) to push to the play store. It also supports pushing to [testing tracks](https://developers.google.com/android-publisher/tracks). Maybe that is useful for you as a building block to make builds before/after qa available with little effort.

**Comment by oliverlockwood** [comment]
@dekarl @jf1 (belatedly, and if you haven't already worked it out)
1.  For a while now we have been publishing updates to a beta track on the Play store.
2.  This issue (making the app available e.g. in DE) is blocked on #46.

**Comment by mvl22** [comment]
This is basically a duplicate of #137, so I'm going to shift further discussion there, and clarify it there.

**Comment by mvl22** [comment]
> For a while now we have been publishing updates to a beta track on the Play store.

https://twitter.com/andrew_shadura/status/1252688960343736323

suggests that the Play Store has the same geographical limitation. Is that correct? Your message above suggests that restriction goes in the beta programme.

**Comment by mvl22** [comment]
> suggests that the Play Store has the same geographical limitation

Update: Seems that was a setting in the Beta program part of the Google Console. I've now made the Beta channel available worldwide.

**Comment by andrewshadura** [comment]
Thanks, I’ll wait for this setting to propagate (it usually takes hours).
